# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/service/VRService.h
# Commit: 898ed02804fe
# Full Hash: 898ed02804fefc9786112f135b9621cc444e6860
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-23 15:52:15
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r e1b7abc99ae9 -r 898ed02804fe gfx/vr/service/VRService.h
--- a/gfx/vr/service/VRService.h	Wed May 22 12:44:04 2019 +0000
+++ b/gfx/vr/service/VRService.h	Wed May 22 22:27:37 2019 +0000
@@ -66,10 +66,7 @@
 #if defined(XP_WIN)
   HANDLE mMutex;
 #endif
-  // We store the value of gfxPrefs::VRProcessEnabled() in mVRProcessEnabled.
-  // This allows us to read the value in the VRService destructor, after
-  // gfxPrefs has been shut down.  We should investigate why gfxPrefs
-  // is shutting down earlier - See bug xxx
+  // We store the value of StaticPrefs::VRProcessEnabled() in mVRProcessEnabled.
   bool mVRProcessEnabled;
 
   bool IsInServiceThread();